apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -1,14 +1,76 @@
+import { $, $$, browser, By, ExpectedConditions } from 'protractor';
```

Comment:
There are tons of await keyword I can see you have used here.
We don't need them. Pls modify this test case.